.github/workflows/cd.yml

```diff
@@ -0,0 +1,60 @@
+name: CD
+
+# on:
+#   push:
+    # Uncomment and add the necessary branches to enable automatic deployment on AWS
+    # branches:
+      # - main
+
+jobs:
+  deploy:
+    name: Deploy on AWS ECS
+    runs-on: ubuntu-latest
+    environment:
+      name: ${{ github.ref_name }}
```

Comment:
If I understand correctly, this requires us to create a Github environment and a branch with the exact same name. If we are forcing any naming convention, we should add it to the docs

Reply:
I'm also wondering if we should keep track of all the releases using the "Github releases" feature, and I think that relies on tagging instead of branch pushes
- https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository
- https://github.com/marketplace/actions/create-a-release-in-a-github-action

Reply:
Using "GitHub releases" based on tagging could be an interesting idea to track all releases. It offers a structured approach with release notes and contributors. However, it requires a different setup from the process investigated in this PR. Let's consider exploring this in a separate card and PR for future enhancements.